spec/q-spec.js

```diff
@@ -1219,8 +1231,11 @@ describe("any", function() {
           .then(function() {
               expect(promise.isRejected()).toBe(true);
               expect(promise.inspect().reason).toBe(expectedError);
-              expect(promise.inspect().reason.message)
-              .toBe("Q can't get fulfillment value from any promise, all promises were rejected. Last error message: Rejected");
+
+              if (!expectNull) {
```

Comment:
Right now Q is rejecting the promise with an undefined/null value. Just because the library user did that doesn't mean that we should as well :)

How about something like:

```
if (expectNull) {
  expect(promise.inspect().reason.message)
  .toBe("Q can't get fulfillment value from any promise, all promises were rejected. Last rejection was null/undefined.");
} else {
  expect(promise.inspect().reason.message)
  .toBe("Q can't get fulfillment value from any promise, all promises were rejected. Last error message: Rejected");
}